src/web/utils/textHelpers.spec.ts

```diff
@@ -96,15 +99,39 @@ describe('Text helper tests', () => {
     });
   });
 
-  describe('sortstrings', () => {
+  describe('sort strings alphabetically', () => {
     const testStringArr = [
       ['atest', 'btest', 'ctest'],
       ['btest', 'atest', 'ctest'],
       ['ctest', 'btest', 'atest'],
     ];
     it.each(testStringArr)('should return array sorted alphabetically', () => {
       for (const t of testStringArr) {
-        expect(sortStrings(t)).toEqual(['atest', 'btest', 'ctest']);
+        expect(sortStringsAlphabetically(t)).toEqual(['atest', 'btest', 'ctest']);
+      }
+    });
+
+    const testStringArrNumbers = [['1test', '7test', '10test']];
+    it.each(testStringArrNumbers)('should return array sorted alphabetically', () => {
+      for (const t of testStringArrNumbers) {
```

Comment:
I don't think this `for` loop is necessary if you are using `it.each`, as that should already do the loop for you. Try something like:
```
const testStringArrays = [
  ['atest', 'btest', 'ctest'],
  ['btest', 'atest', 'ctest'],
  ['ctest', 'btest', 'atest'],
];
const expected = ['atest', 'btest', 'ctest'];
testStringArrays.forEach((testStringArray) => {
  it('should return an alphabetically sorted array', () => {
    expect(sortStringsAlphabetically(testStringArray)).toEqual(expected);
  });
});
```